services/netlify/netlify.service.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const { renderBuildStatusBadge } = require('../build-status')
+const { BaseSvgScrapingService } = require('..')
+
+const pendingStatus = 'building'
+const notBuiltStatus = 'not built'
+
+const statusMap = {
+  building: pendingStatus,
+  waiting: pendingStatus,
+  initiated: pendingStatus,
+  stopped: notBuiltStatus,
+  ignored: notBuiltStatus,
+  blocked: notBuiltStatus,
+  infrastructure_failure: 'failed',
+}
+
+module.exports = class Netlify extends BaseSvgScrapingService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'netlify',
+      pattern: ':projectId',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Netlify',
+        pattern: ':projectId',
```

Comment:
You can remove this line. You can use `pattern` in an example if you need to do something different than the route pattern, but no need to redfine pattern here if it's the same

Reply:
This `pattern` used to be different before https://github.com/badges/shields/pull/3953/commits/4369658193b48011d582701ae01d96d8b1916c26, which changed the one in `route`.